README.md

```diff
@@ -1,13 +1,22 @@
-- [The Three Methods](#the-three-methods)
-  - [Mutable `copy`](#mutable-copy)
-  - [Mapping `copyMap`](#mapping-copymap)
-  - [Value class `copy`](#value-class-copy)
-  - [`copy` for hierarchies](#copy-for-hierarchies)
-- [Using KopyKat in your project](#using-kopykat-in-your-project)
-  - [Customizing the generation](#customizing-the-generation)
-- [What about optics?](#what-about-optics)
-
-One of the great features of Kotlin [data classes](https://kotlinlang.org/docs/data-classes.html) is their [`copy` method](https://kotlinlang.org/docs/data-classes.html#copying). But using it can become cumbersome very quickly, because you need to repeat the name of the field before and after.
+<!-- TOC -->
+
+* [What can KopyKat do?](#what-can-kopykat-do)
+    * [Mutable `copy`](#mutable-copy)
+        * [Nested mutation](#nested-mutation)
+    * [Mapping `copyMap`](#mapping-copymap)
+    * [Value classes](#value-classes)
+        * [The `copyMap` function](#the-copymap-function)
+        * [The `copy` function](#the-copy-function)
+    * [`copy` for sealed hierarchies](#copy-for-sealed-hierarchies)
```

Comment:
Would you mind moving this one on top of the value classes one? I feel like this is more interesting to most people.

Reply:
Yeah, I was actually contemplating making the value class a note on the relevant section. What do you think?

Reply:
That sounds reasonable. At the end of the day, values classes are the least used of all the things we support.

Reply:
There you go :) fc172ad